examples/tracking_pycaret.py

```diff
@@ -0,0 +1,218 @@
+"""
+About
+
+Use MLflow and CrateDB to track the metrics, parameters, and outcomes of an ML
+experiment program using PyCaret. It uses the Real-world sales forecasting benchmark data
+dataset from 4TU.ResearchData.
+
+- https://github.com/crate-workbench/mlflow-cratedb
+- https://mlflow.org/docs/latest/tracking.html
+
+Usage
+
+Before running the program, define the `MLFLOW_TRACKING_URI` environment
+variable, in order to record events and metrics either directly into the database,
+or by submitting them to an MLflow Tracking Server.
+
+    # Use CrateDB database directly
+    export MLFLOW_TRACKING_URI="crate://crate@localhost/?schema=mlflow"
+
+    # Use MLflow Tracking Server
+    export MLFLOW_TRACKING_URI=http://127.0.0.1:5000
```

Comment:
If we extensively talk about how to configure `MLFLOW_TRACKING_URI` here, we should be fair and also do the same with `CRATEDB_HTTP_URL`, which is used a few lines below.

Reply:
Hmm now that I think about it: should we really talk about this env var? 
The default value works for the sake of the example. 
But the MLFLOW envs need to be set.
As you are the originator of this code snippet and I simply built upon it, what would you suggest?